protocol/v2/ssv/value_check.go

```diff
@@ -68,11 +73,22 @@ func BeaconVoteValueCheckF(
 			Target:          bv.Target,
 		}
 
+		var slashableValidatorPubKeys []spectypes.ShareValidatorPK
 		for _, sharePublicKey := range sharePublicKeys {
 			if err := signer.IsAttestationSlashable(sharePublicKey, attestationData); err != nil {
-				return err
+				logger.Warn("attestation is slashable, skipping",
+					fields.Validator(sharePublicKey[:]), fields.Slot(slot), zap.Error(err))
+				slashableValidatorPubKeys = append(slashableValidatorPubKeys, sharePublicKey)
+				continue
 			}
 		}
+
+		// If all validators are slashable, return an error
+		if len(slashableValidatorPubKeys) == len(sharePublicKeys) {
+			return fmt.Errorf("all validators are slashable, attestation cannot proceed")
+		}
+
+		//If at least one validator is not slashable, return nil
 		return nil
 	}
 }
```

Comment:
@GalRogozinski wdyt about this?
 
i think one of the issues with our solution is that we will proceed to try to do the duty for the slashable validator (and later fail of course) which seems inefficient (which i hope is the only problem)

@liorrutenberg suggested that we should instead just do our own check before the ValueCheck to remove the slashable validators from the list and also from the Runner's duties (so it won't try to do them), however i think that's a bit complex to pull of because ValueCheck happens again with every round change right? (and not only on duty start which is easy for us to filter out slashables there)

Reply:
Hey,
I think you shouldn't make new validators slashable and that's it.
If you don't want to run new validators in the beginning add a new flag for them.
When we did the spec we were aware of this issue. It was decided to look into it later after Alan fork. I created a Monday task but it never managed to get enough priority.

Slashing should be looked out in an orderly way by the spec team and not with a hasty hotfix

Reply:
@GalRogozinski new validators is not the only reason this can happen

we run committee runners for each slot in parallel, so some old runner at a late round might have suddenly one slashable validator because that validator already signed a newer duty in a newer runner

in that case we'd be failing the entire slot for all other validators as well

regardless of the fact that we should indeed just avoid starting validators in the epoch where they registered (which we'll do in a future release), this is still a problem

Reply:
@moshe-blox 
This was known... the reason I agreed to have it done this way is because I was afraid that there may be subtle attack-vectors on the other way. I don't remember the details now. I am also not sure if we should worry about them now.

Slashing is the most crucial part of the system, we shouldn't just touch it this way.

How often this happens w.o new validators? Is it such a huge issue that we can't wait with this?

Reply:
@moshe-blox @GalRogozinski 
New validators and their duty execution are not related to consensus.
If the node decided not to start a duty on registration epoch this should be handled there and not to start consensus for it and check it there

Reply:
>  @GalRogozinski @olegshmuelov  wdyt about this?
>  
> i think one of the issues with our solution is that we will proceed to try to do the duty for the slashable validator (and later fail of course) which seems inefficient (which i hope is the only problem)
> 
> @liorrutenberg suggested that we should instead just do our own check before the ValueCheck to remove the slashable validators from the list and also from the Runner's duties (so it won't try to do them), however i think that's a bit complex to pull of because ValueCheck happens again with every round change right? (and not only on duty start which is easy for us to filter out slashables there)

I guess we could return the slashable ones and drop them but that changes the whole flow so not suitable for a Hotfix..